frontend/src/components/RoleIcons.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { ReactComponent as BusinessValueFilled } from '../icons/rate_value_filled.svg';
+import { ReactComponent as RequiredWorkFilled } from '../icons/rate_work_filled.svg';
+import css from './RoleIcons.module.scss';
+
+const classes = classNames.bind(css);
+
+interface IconProps {
+  size?: 'xxsmall' | 'xsmall' | 'small' | 'default' | 'large';
+  color?: string;
+}
+
+const svgIcon = (Icon: FC, { size, color }: IconProps) => (
```

Comment:
```suggestion
const svgIcon = (Icon: FC): FC<IconProps> => ({ size, color }) => (
```

This could be curried, allowing simpler use:

```ts
export const BusinessIcon = svgIcon(BusinessValueFilled);
export const WorkRoundIcon = svgIcon(RequiredWorkFilled);
```